tools/gltfViewer/rotation.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
Why this file?

Reply:
oops, it's part of the kuesa template I used to initially create the project, will remove